5.0/en/0x50-V50-attacks-via-browser.md

```diff
@@ -1,32 +1,13 @@
-# V50 Defense against attacks via browser
+# V50 Browser-based Attacks
```

Comment:
This seems like the most accurate definition for now

Reply:
For me "Browser-based" sounds like someone is using vulnerability in browser itself...

Reply:
Maybe you prefer Browser-based Protections?

Reply:
Will change to web frontend as agreed on a call